pkg/apis/pipeline/v1/param_types.go

```diff
@@ -195,9 +195,9 @@ func (ps Params) extractParamMapArrVals() map[string][]string {
 // Params is a list of Param
 type Params []Param
 
-// extractParamArrayLengths extract and return the lengths of all array params
+// ExtractParamArrayLengths extract and return the lengths of all array params
 // Example of returned value: {"a-array-params": 2,"b-array-params": 2 }
-func (ps Params) extractParamArrayLengths() map[string]int {
+func (ps Params) ExtractParamArrayLengths() map[string]int {
```

Comment:
since this is now exported, shouldn't it have its own unit tests?

Reply:
alternatively, we can remove the receiver and turn this into a regular function and move it to the package its being used in?

Reply:
added tests!